.github/ISSUE_TEMPLATE/issue_template.md

```diff
@@ -0,0 +1,29 @@
+### Which part of the code is the issue addressing?
+
+* [ ] Archicture 
+* [ ] Adressing a bug
+* [ ] Refactoring
+* [ ] pallet_xyz
+* [ ] pallet_xyz
+* [ ] Suggesting a new pallet
+* [ ] Runtime
+* [ ] Cross-Chain
+* [ ] Protocol
```

Comment:
I would keep the list shorter. Remove the circles, as they are not constant and probably unknown to person not familiar with the company. We can manually add the labels for the circles. WDYT?

I would keep 
* Addressing a bug
* Refactoring
* Feature